### DISCUSSION TITLE
Calendar and emails

### DISCUSSION THREAD

**Discussion by mcDcall** [twentyhq/twenty#8777] [Engineering vision]
I'm sorry for the maybe stupid questions...
Can i setup a caldav calendar? I have a nextcloud instance and i would like to set up it's calendar in twenty.
Can i create a calendar? If i can't setup a calendar besides google, can i create one? I see a lot of Rest api endpoints...
Can i setup my company email in twenty? How? (I only see to connect google account)

Thanks in advance for your reply.

**Comment by gonzaleztroyano** [comment]
Related to  https://github.com/twentyhq/twenty/discussions/11701

**Comment by BOHEUS** [comment]
@mcDcall now it's possible to connect CalDav calendar to Twenty in Settings > Calendar, it should be enabled by default
